### DISCUSSION TITLE
Need documentation

### DISCUSSION THREAD

**Discussion by MHAMZA852** [Ansarimajid/College-ERP#9] [Q&A] [answered]
i need documentation of this project in pdf or docx , in which all detail database , what project do, each and every , can you give me please

**Comment by MHAMZA852** [comment]
like this 
kindly check this link 
https://github.com/samarth-p/College-ERP/blob/master/SE%20report%20final.pdf

**Comment by Ansarimajid** [comment, marked as answer]
Here's short [Mini Project Report](https://github.com/Ansarimajid/College-ERP/blob/811cce08d8f8d9c9d261fa764f7c5c21fa796d53/A18_MINI%20PROJECT%20REPORT.pdf)

**Comment by MHAMZA852** [comment]
Thank you so much
